backup/util/loggers/base_logger.class.php

```diff
@@ -114,7 +114,8 @@ public function is_checksum_correct($checksum) {
 
     public final function process($message, $level, $options = null) {
         $result = true;
-        if ($this->level != backup::LOG_NONE && $this->level >= $level) { // Perform action conditionally
+        if ($this->level != backup::LOG_NONE && $this->level >= $level
+            && !(defined('BEHAT_TEST') && BEHAT_TEST)) { // Perform action conditionally.
```

Comment:
:lipstick: can we double indent this to make it clear that line 118 is part of the condition and not the branch?

Reply:
:heavy_check_mark: